### PR TITLE
Ds user profile

### PR DESCRIPTION
**Pull request by dylansarikas**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'users/show'
```
Comment (APatel-AI) [inline config/routes.rb]: Lines 2 and 3 can be deleted because `get "/:username" => "users#show", as: :user` already handles that action.

**Review by APatel-AI — CHANGES_REQUESTED** [review]
Great job Dylan!!. I have one minor additional comment for the routes section.

File: lib/tasks/dev.rake [hunk 2]
```diff
@@ -37,16 +37,16 @@ task :sample_data => :environment do
...
           recipient: second,
           status: FollowRequest.statuses.values.sample
         )
       end
 
-      if rand < 0.667
-        first.sent_follow_requests.create(
-          recipient: second,
+      if rand < 0.7
+        second.sent_follow_requests.create(
```
Comment (AdolfoNava) [inline lib/tasks/dev.rake:48]: Good correction on the nested loop here.

**Review by AdolfoNava — APPROVED** [review]

**Review by APatel-AI — APPROVED** [review]
Looks good

File: app/views/comments/_form.html.erb [hunk 3]
```diff
@@ -11,22 +11,13 @@
...
-  <div>
-    <%= form.label :author_id, style: "display: block" %>
-    <%= form.text_field :author_id %>
-  </div>
-
-  <div>
-    <%= form.label :photo_id, style: "display: block" %>
-    <%= form.text_field :photo_id %>
-  </div>
+  <%= form.hidden_field :photo_id %>
```
Comment (JoseMDO) [inline app/views/comments/_form.html.erb:14]: Nice job using hidden inputs!

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        <div class="card-body">
+          <h5 class="card-title"> <%= photo.owner.username %> </h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
```
Comment (JoseMDO) [inline app/views/users/show.html.erb:22]: If we dont need these empty image tags, its good to either add default values or remove them so they dont create accessibility issues :)

File: config/routes.rb [hunk 5]
```diff
@@ -7,5 +7,8 @@
   resources :follow_requests
   resources :likes
   resources :photos
+  #resources :users, only: :show
   
+  get "/:username" => "users#show", as: :user
```
Comment (JoseMDO) [inline config/routes.rb:12]: Nice

**Review by JoseMDO — APPROVED** [review]
Looks good Dylan!

**Pull request merged**